README.md

```diff
@@ -1,88 +1,97 @@
 ![Docker](https://img.shields.io/badge/trueblocks-docker-blue.svg)
 [![Twitter](https://img.shields.io/twitter/follow/espadrine.svg?style=social&label=Twitter)](https://twitter.com/quickblocks?lang=es)
 
-The dAppNode project answers the "how" question for running an Ethereum node. TrueBlocks answers the "why" question. dAppNode makes setting up and running a node easier. TrueBlocks makes extracting useful data ***from your particular list of accounts*** from that node easier.
+TrueBlocks extracts address appearances from an Ethereum chain and creates a lightweight index, making queries by address a trustless and fast possibility.
 
-## Requirements
+trueblocks-docker is a docker image containing an installation of TrueBlocks, and an http API server that delivers Ethereum data by address.
 
-- A running Parity Ethereum node with `--tracing on` enabled. You can do this on the  [DNP_ETHCHAIN](https://github.com/dappnode/DNP_ETHCHAIN) package using `EXTRA_OPTS=--tracing on` ([Why do we need --tracing enabled?]())
-- An additional 100GB of hard drive space for the TrueBlocks caches.
-- Time (it takes a long time to sync the TrueBlocks caches: [why?]()).
+<!-- By visiting every bit of the Ethereum data including blocks, transactions, receipts, logs, traces, trace actions, and trace results for every block, TrueBlocks is able to extract every 'appearance' of an address on the blockchain.
 
-## Usage examples
+Trueblocks finds nearly 30% more appearances than naive approaches such as EtherScan. Sorting the list and creating an index allows TrueBlocks to reconstruct historical lists of transactions per address more than 150,000 times faster than a raw scan of the chain via the RPC.
 
-- **You are already running an Parity Ethereum node with `--tracing` enabled:**
-    - Start the TrueBlocks docker container with `docker-compose up -d`
-    - Wait for TrueBlocks to build its cache. ([Why does this take so long?](wiki/why-is-trueblocks-so-slow))
-    - [Query TrueBlocks for a list of transactions on your accounts](wiki/tutorial-querying-for-lists-of-transactions-per-account)
-    - Given the list of transactions, [query the node for full JSON details of those transactions](wiki/tutorial-querying-for-lists-of-transactions-per-account)
-- **You're not running a node:** Use dAppNode to run a Parity node with tracing enabled. Follow instructions above...
-- **You're running geth:** Use dAppNode to run a Parity node with tracing enabled. Follow instructions above...
-- **You don't have --tracing enabled:** [Enable --tracing](wiki/enabling-tracing-on-a-dAppNode), resync the node, and follow instructions above...
+The system runs on commercial hardware (laptops), and is able to keep up to the chain. This solves the problem posed by web-scale API-based data providers such as EtherScan who threaten to capture Ethereum’s blockchain data to the detriment of the entire community. -->
```

Comment:
Again, maybe we can remove "such as EtherScan" and just leave it as "...posed by web-scale API-based data providers..."

Reply:
Agree. This section is commented out right now. Let's come back to it in our next iteration.